### ISSUE TITLE
Discrepancy between reproduced results and reported performance

### ISSUE THREAD

**Issue by lulu-zii** [kk42yy/CoStoDet-DDPM#3]
Thanks for your great contribution. I am writing to report an issue I encountered while trying to reproduce the results on the AutoLaparo dataset.

Following the instructions in the README.md. However, after training for [50] epochs, the best accuracy I achieved was around 85.6%, which is lower than the 87.6% reported in the paper.

<img width="879" height="108" alt="Image" src="https://github.com/user-attachments/assets/f5dc5d29-9ff4-493a-9f67-34fa6e9cef74" />

I would like to ask if there are any specific hyperparameters or hidden tricks (e.g., specific environment versions, hardware dependencies, or random seeds) that are essential for reproducing the reported performance?

Any guidance or suggestions would be greatly appreciated.

**Comment by kk42yy** [comment]
Dear yingzhu,

This project was completed some time ago, and the information below is provided based on our understanding and may be helpful as a reference.

* **Hardware**: 4 × NVIDIA RTX 4090 GPUs
* **Environment**: Install dependencies as follows (see [requirements.txt](https://github.com/kk42yy/CoStoDet-DDPM/blob/main/Cholec80/requirements.txt) for others):

  ```bash
  pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchaudio==0.13.1 \
    --extra-index-url https://download.pytorch.org/whl/cu117
  ```
* **Randomness**: We have trained using random seeds.

#### Empirical Notes

* For the **two-stage training**, if stage one performs well, consider a smaller learning rate (e.g., `1e-5` or lower), freezing more parameters, and fewer epochs (e.g., $\le 15$) in stage two; otherwise, try a larger learning rate, fewer frozen parameters, and more epochs.
* Backbone choice may affect results; **ConvNeXt V1-T** or other variants can be explored.

Best wishes,
Kaixiang

**Comment by lulu-zii** [comment]
ok, thanks
